ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-gas-limit/advanced-gas-fee-gas-limit.js

```diff
@@ -40,17 +43,20 @@ const AdvancedGasFeeGasLimit = () => {
     setGasLimitInContext(gasLimit);
     const error = validateGasLimit(gasLimit, minimumGasLimitDec);
     setGasLimitError(error);
-  }, [gasLimit, minimumGasLimitDec, setGasLimitInContext]);
+    setErrorValue('gasLimit', error === 'editGasLimitLessThanMinimum');
+  }, [gasLimit, minimumGasLimitDec, setGasLimitInContext, setErrorValue]);
+
+  const errorMessage =
+    gasLimitError &&
+    (gasLimitError === 'editGasLimitLessThanMinimum'
+      ? t(gasLimitError, [minimumGasLimitDec - 1])
+      : t(gasLimitError, [MAX_GAS_LIMIT_DEC]));
```

Comment:
[nit] I feel like this is a bit confusing with the `&&` chaining to a ternary
```suggestion
let errorMessage = ''
if(gasLimitError){
  errorMessage = gasLimitError === 'editGasLimitLessThanMinimum'
      ? t(gasLimitError, [minimumGasLimitDec - 1]
      : t(gasLimitError, [MAX_GAS_LIMIT_DEC]);
 }
```
just a personal preference for legibility